src/main/kotlin/no/nav/familie/ef/personhendelse/inntekt/FinnPersonerFyltTjueFemArbeidsavklaringspengerTask.kt

```diff
@@ -0,0 +1,69 @@
+package no.nav.familie.ef.personhendelse.inntekt
+
+import com.fasterxml.jackson.module.kotlin.readValue
+import no.nav.familie.ef.personhendelse.client.pdl.PdlClient
+import no.nav.familie.kontrakter.felles.objectMapper
+import no.nav.familie.prosessering.AsyncTaskStep
+import no.nav.familie.prosessering.TaskStepBeskrivelse
+import no.nav.familie.prosessering.domene.Task
+import no.nav.familie.prosessering.internal.TaskService
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
+import org.springframework.stereotype.Service
+import java.time.YearMonth
+
+@Service
+@TaskStepBeskrivelse(
+    taskStepType = FinnPersonerFyltTjueFemArbeidsavklaringspengerTask.TYPE,
+    maxAntallFeil = 1,
+    settTilManuellOppfølgning = true,
+    beskrivelse = "Finner personer med som mottar arbeidsavklaringspenger og har fylt 25 den siste måneden.",
+)
+class FinnPersonerFyltTjueFemArbeidsavklaringspengerTask(
+    private val pdlClient: PdlClient,
+    private val taskService: TaskService,
+) : AsyncTaskStep {
+    val logger: Logger = LoggerFactory.getLogger(this::class.java)
+    val secureLogger: Logger = LoggerFactory.getLogger("secureLogger")
+
+    override fun doTask(task: Task) {
+        val (inntektsendringForBrukereMedArbeidsavklaringspenger, årMåned) = objectMapper.readValue<PayloadFinnPersonerFyltTjueFemArbeidsavklaringspengerTask>(task.payload)
+        val startDato = årMåned.minusMonths(1).atDay(6)
+        val sluttDato = årMåned
```

Comment:
Variabelen kan enten fjernes, eller så kan du sette
```suggestion
        val sluttDato = årMåned.atDay(7)
```